README.md

```diff
@@ -59,11 +59,17 @@ However, if you wish to run your own static server, there are several options av
 
 Assuming you have Bramble running on port `8000`. Now you can visit [http://localhost:8000/src](http://localhost:8000/src).
 
+**NOTE 1:** Bramble expects to be run in an iframe, which hosts its filesystem. For local
```

Comment:
given that bramble already tells you this with a link to the correct file, I don't think we need to explicitly point this out in the README.md... having a convenient click through when opening localhost:8000 feels convenient enough.